### PR TITLE
Add a simple collector uninstalling step docs

### PR DESCRIPTION
**Pull request by keiko713**
It's super simple one, but good enough as a starter

### CODE REVIEW

File: collector/uninstalling.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+```
+sudo yum remove pganalyze-collector
+```
+
+Then, clean up a state file and its directory:
+```
+sudo rm -r /var/lib/pganalyze-collector
+```
+
+Finally, clean up the user and group:
```
Comment (msakrejda) [inline collector/uninstalling.mdx:48]:
```suggestion
Finally, clean up the collector system user and group:
```
I think clarifying "system" user here makes it clearer what you're doing. It should be pretty obvious from the `DROP USER` above, but maybe the reader skipped past that section and jumped right into this.

File: collector/uninstalling.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+On systems using Debian-based packages, you can uninstall the collector through
+the standard uninstall mechanism.
+
+First, remove the collector package from your system (this includes disabling
+and stopping the service):
+```
+sudo apt remove pganalyze-collector
+```
+
+Then, clean up a state file and its directory:
```
Comment (msakrejda) [inline collector/uninstalling.mdx:64]: Same as above.

File: collector/uninstalling.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+```
+sudo rm -r /var/lib/pganalyze-collector
+```
+
+Finally, clean up the user and group:
```
Comment (msakrejda) [inline collector/uninstalling.mdx:69]: Same as above.

**Review by msakrejda — APPROVED** [review]
Nice, this is great. I suggested "remove" or "drop" in some places, since "clean up" is a little vague, and when you're doing a specific action, we can talk about it more concretely.

File: collector/uninstalling.mdx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+2. [Uninstalling the collector](#uninstalling-the-collector)
+
+## Cleaning up a monitoring user and helper functions
+
+To uninstall the pganalyze collector, first you'll need to clean up a pganalyze
+monitoring user and helper functions that you created during the setup.
+
+* Run `DROP SCHEMA pganalyze CASCADE` in each database that you monitor to
+  remove helper functions
+* Run `DROP USER pganalyze` in any database to remove a monitoring user
```
[the remark this answers is not in the document]
Comment (keiko713) [reply collector/uninstalling.mdx:19]: Yeah exactly. I think there are other things that are customizable (e.g. the location of the config/state file, etc.) but I didn't call them out here as I wanted to keep this doc simple.

**Pull request merged**